build.cake

```diff
@@ -20,7 +22,17 @@ var OWNERS = new [] { "Charlie Poole" };
 var DESCRIPTION = "This extension allows NUnit to recognize and load NUnit projects, file type .nunit.";
 var SUMMARY = "NUnit Engine extension for loading NUnit projects.";
 var COPYRIGHT = "Copyright (c) 2016 Charlie Poole";
-var RELEASE_NOTES = new [] { "See https://raw.githubusercontent.com/nunit/nunit-project-loader/master/CHANGES.txt" };
+var RELEASE_NOTES = new [] { 
```

Comment:
This will add another manual step to each release. Is leaving the link perhaps preferable?

Alternatively, if we gave the changes.txt into changes.md, and formatted it with headers as nunit/nunit-console, it would then be trivial the read that file as far as the second main header, and include the appropriate release notes.